app/models/gsa18f/procurement.rb

```diff
@@ -19,10 +19,12 @@ def self.purchase_amount_column_name
     include ProposalDelegate
     include PurchaseCardMixin
 
+    validates :cost_per_unit, presence: true
     validates :quantity, numericality: {
       greater_than_or_equal_to: 1
-    }
+    }, presence: true
     validates :product_name_and_description, presence: true
+    # TODO add conditional validations for recurring charges
```

Comment:
What about putting it in tracker!? 

Totally makes sense to do things separately, but because there is so little room for context here it becomes a to do for *you* rather than anyone on the team who might want to pitch in - LET US HELP (it may be the case that others could guess which conditional validations you are referring to, but I am so new here that I really have no idea)